src/applications/discharge-wizard/components/v2/resultsComponents/StepTwo.jsx

```diff
@@ -143,6 +143,63 @@ const StepTwo = ({ formResponses }) => {
     default:
   }
 
+  const renderBuddyStatements = () => {
+    return (
+      <li>
+        <strong>
+          Buddy statements (also called a Statement in Support of Claim or VA
```

Comment:
The ticket says this should be an `<h4>` - is that right?

<img width="629" alt="Screenshot 2024-11-15 at 3 07 35 PM" src="https://github.com/user-attachments/assets/a40b7dca-df96-4b89-ac0c-7402a153861f">